react/features/poll/components/PollsList/web/CreatePollButton.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { openPollCreationPage } from '../../../actions';
+import {
+    FooterButton
+} from '../../PollPanel/styled';
+
+export function CreatePollButton() {
+    const dispatch = useDispatch();
+    const { t } = useTranslation();
+    const showPollList = useSelector(state => state['features/poll'].showPollList);
```

Comment:
Do we need to specifically have a `showPollList` state? We have an `isOpen` state for opening and closing the pane and the `pollPaneMode` state to determine what pane to show.